pkg/serving/config_changes.go

```diff
@@ -226,3 +240,7 @@ func updateAnnotations(annotations map[string]string, toUpdate map[string]string
 	}
 	return nil
 }
+
+func isValidPullPolicy(policy string) bool {
+	return policy == string(v1.PullAlways) || policy == string(v1.PullNever) || policy == string(v1.PullIfNotPresent)
+}
```

Comment:
Does it need to be case sensitive (capital letter and camel case in the last `IfNotPresent`)? `--pull-policy always` seems reasonable.
I'd go with something like `getPullPolicy` to check all lower/upper and return `v1.PullPolicy` or empty/nil otherwise?

The other option would be 3 standalone `--pull-policy-*`, but that's too much clutter probably. :)

Reply:
Yes, that makes sense. I made the change to make the flag case insensitive.